string raw_action_type
string action_type
eager_core/Object[] observation_objects
---
eager_core/Space action_space
